src/ert/analysis/_es_update.py

```diff
@@ -347,17 +355,32 @@ def _load_observations_and_responses(
 
     ens_std_mask = ens_std > std_cutoff
     ens_mean_mask = abs(observations - ens_mean) <= alpha * (ens_std + errors)
-
     obs_mask = np.logical_and(ens_mean_mask, ens_std_mask)
 
-    update_snapshot = UpdateSnapshot(
-        obs_name=obs_keys,
-        obs_value=observations,
-        obs_std=errors,
-        obs_mask=obs_mask,
-        response_mean=ens_mean,
-        response_std=ens_std,
-    )
+    update_snapshot = []
+    for name, obs_val, obs_std, status, mean, std, mean_mask, std_mask in zip(
```

Comment:
Are these name-changes necessary? I find them a bit confusing.

Reply:
Compared with the class? Can align it to them. Not very fond of this huge zip, am happy if you have alternatives.